src/components/content/order/formElements/NumberInput.tsx

```diff
@@ -5,23 +5,40 @@
 
 import { NumberInputEventHandler, DeployParam } from './CommonTypes';
 import { Form, InputNumber } from 'antd';
+import React from 'react';
+import { Rule } from 'rc-field-form/lib/interface';
 
 export function NumberInput({
     item,
     onChangeHandler,
 }: {
     item: DeployParam;
     onChangeHandler: NumberInputEventHandler;
-}): JSX.Element {
+}): React.JSX.Element {
+    const ruleItems: Rule[] = [{ required: item.mandatory }, { type: 'number' }, { min: 0 }];
+    let minimum: number;
+    let maximum: number;
+
+    const setValidator = (_: unknown, value: number) => {
+        if (value <= minimum) {
+            return Promise.reject(new Error(`The new password not match minimum ${minimum}!`));

```

Comment:
it is not always a password field. We need a better error message here.
`Value does not satisfy criteria minimum ${minimum}`